physics/speeds_of_gas_molecules.py

```diff
@@ -0,0 +1,110 @@
+"""
+The root-mean-square, average and most probable speeds are derived from
+the Maxwell-Boltzmann distribution. The Maxwell-Boltzmann distribution is a
+probability distribution that describes the distribution of speeds for particles
+in a gas.
```

Comment:
```suggestion
The root-mean-square, average and most probable speeds of gas molecules
are derived from the Maxwell-Boltzmann distribution. The Maxwell-Boltzmann
distribution is a probability distribution that describes the distribution of
speeds of particles in an ideal gas.
```
A few minor additions for clarity